Resources/Prototypes/whitelists.yml

```diff
@@ -0,0 +1,38 @@
+﻿- type: PlayerConnectionWhitelist
```

Comment:
Separate the MRP template into another file and rename it so that it's clear that it's wizden-specific policy that downstreams may not want to use.